src/pymovements/plotting/traceplot.py

```diff
@@ -21,12 +21,17 @@
 from __future__ import annotations
 
 import sys
+from typing import Dict
+from typing import Literal
+from typing import Sequence
+from typing import Tuple
 
-import matplotlib
+import matplotlib.colors
```

Comment:
this change follows: https://github.com/matplotlib/matplotlib/issues/24950#issuecomment-1379944582